app/recipe/views.py

```diff
@@ -0,0 +1,41 @@
+from rest_framework import viewsets, mixins
+
+from core.models import Recipe, Ingredient
+
+from recipe import serializers
+
+
+class IngredientViewSet(viewsets.GenericViewSet, mixins.ListModelMixin):
+    """Manage recipes in the database"""
+    serializer_class = serializers.IngredientSerializer
+    queryset = Ingredient.objects.all()
+
+    def get_serializer_class(self):
+        """Return serializer class"""
+        return self.serializer_class
+
+    def perform_create(self, serializer):
+        """Create new ingredient"""
+        serializer.save()
+
+
+class RecipeViewSet(viewsets.ModelViewSet):
+    """Manage recipes in the database"""
+    queryset = Recipe.objects.all()
+    serializer_class = serializers.RecipeSerializer
+
+    def get_serializer_class(self):
+        """Return serializer class"""
+        return self.serializer_class
+
+    def perform_create(self, serializer):
+        """Create new recipe"""
+        serializer.save()
```

Comment:
Those methods are the same as in the superclass here, right? Why do you need to define them again?

Reply:
Correct, I had left this here as a reminder to myself in case I needed it.  Should've been removed for the PR.